src/main/java/com/faforever/client/chat/UserFilterController.java

```diff
@@ -209,19 +209,13 @@ private void updateGameStatusMenuText(PlayerStatus status) {
       return;
     }
 
-    switch (status) {
-      case PLAYING:
-        gameStatusMenu.setText(i18n.get("game.gameStatus.playing"));
-        break;
-      case LOBBYING:
-        gameStatusMenu.setText(i18n.get("game.gameStatus.lobby"));
-        break;
-      case IDLE:
-        gameStatusMenu.setText(i18n.get("game.gameStatus.none"));
-        break;
-      default:
-        throw new ProgrammingError("Uncovered player status: " + status);
-    }
+    String key =  switch (status) {
+      case PLAYING -> PLAYING.getI18nKey();
+      case LOBBYING -> LOBBYING.getI18nKey();
+      case IDLE -> IDLE.getI18nKey();
+      default -> throw new ProgrammingError("Uncovered player status: " + status);
+    };
+    gameStatusMenu.setText(i18n.get(key));
```

Comment:
You can just use this 

`gameStatusMenu.setText(i18n.get(status.getI18nKey()));`